raspisump/database.py

```diff
@@ -0,0 +1,73 @@
+""" Module to write water level reading to a database."""
+
+# Raspi-sump, a sump pump monitoring system.
+# Al Audet
+# http://www.linuxnorth.org/raspi-sump/
+# Module by: USAF_Pride
+#
+# All configuration changes should be done in raspisump.conf
+# MIT License -- http://www.linuxnorth.org/raspi-sump/license.html
+
+
+import sqlite3
+
+# here just in case
+try:
```

Comment:
This should not be responsible for reading in the config file, pass the full file path into the function